pipelines/ingest-gbif/src/main/java/org/gbif/pipelines/ingest/pipelines/VerbatimToInterpretedPipeline.java

```diff
@@ -136,6 +138,10 @@ public static void run(InterpretationPipelineOptions options) {
     BasicTransform basicTransform =
         BasicTransform.builder()
             .keygenServiceSupplier(KeygenServiceFactory.createSupplier(config, datasetId))
+            .lifeStageLookupSupplier(
+                () ->
+                    VocabularyLookup.load(
+                        FsUtils.readVocabularyFile(hdfsSiteConfig, vocabulariesPath, "LifeStage")))
```

Comment:
The intention here is to move off enums and make use of a DB driven vocabulary server. A community of users can then manage the vocabulary through the UI at registry.gbif.org, and they can then release changes to the vocabulary which will be picked up at the next processing run without requiring software deployment (similar to e.g. the taxonomic lookups).

Reply:
Thanks, Tim.
I understand the current concept of vocabulary, my question was more about to give an enum name to specific vocabulary, not values inside

Reply:
Ah. I'd suggest we probably want to make that a configurable thing at some point too, so that a config change and not a code release and deploy cycle can enable it for simple string fields. E.g. along the lines of

```
  // Fields backed by the vocabulary server for basic parsing and interpretation:
  simpleVocabularyBackedTerms: [
    {"term": "https://rs.tdwg.org/dwc/terms/sex", "vocabulary":"https://registry.gbif.org/..."},
    {"term": "https://rs.tdwg.org/dwc/terms/lifestage", "vocabulary":"https://registry.gbif.org/..."},
    {"term": "https://rs.tdwg.org/dwc/terms/basisOfRecord", "vocabulary":"https://registry.gbif.org/..."}
  ]
```

While this is a single example to prove the whole concept, I'm not sure that is worth it at this stage

Reply:
That was my first thought, In this case, I would use PipelinesConfig and add VocabularyConfig object with path to vocabulary file, life stage vocabulary name, and etc. I believe we will add more variables in the future.

Reply:
I created an enum for now just for the names. Maybe in the future we have to refactor it and create something like what Tim suggests.